Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: all
```

Comment:
@xoxys Isn't it a bit overpowered / duplicated for pure js projects to have a makefile in addition to the `package.json` scripts?